### ISSUE TITLE
Docker self updating

### ISSUE THREAD

**Issue by maddentim** [lando/lando#2917]
In the past, I tried to stay on the version of Docker for Mac that came with Lando. Recently, Docker has been updating itself without asking. Kinda annoying tbh. I am now on docker 3.2.2 and Lando is complaining that I am using an unsupported version. I looked in the Docker setting for a way to tell it to not update, but didn't see anything. I would prefer not to reinstall Lando every day or two when docker has to be restarted. What's a poor guy to do? (I realize you guys didn't cause this but you're so wise and knowing)

**Comment by pirog** [comment]
Our "unsupported version of docker" warning is just to let you know you are beyond what we have tested against. If you are only a few patch versions ahead of what we support then MOST OF THE TIME this is fine. you'll just have to deal with the warning until we increase the range and cut a new version.

**Comment by shstkvch** [comment]
@pirog thanks for this and for your and the team's excellent work on Lando. Do you know if there is a way to turn off Docker auto-updates OR to disable the update nag? It's showing up every day now since Docker started auto-updating

I believe Docker are working on a fix (https://github.com/docker/roadmap/issues/183) but it would be helpful to work around this in the meantime

**Comment by maddentim** [comment]
The one thing I had read (in the docker thread) that seemed like a workaround was to block network connections to desktop.docker.com. Seemed a little extreme and so far lando has been compatible (apparently) with the current version so I have not looked at it. This is probably not something that can be solved by Lando...

**Comment by pirog** [comment]
As of right now we don't intend to change anything on our end. That might change but i think we want to see what docker does first.

**Comment by amanire** [comment]
Noting that latest release changes the behavior, and it just nags you instead, see https://github.com/docker/roadmap/issues/183#issuecomment-818910160 from thread above:

> [Y]ou are never forced to [update], you have the option to dismiss the popup until the next day.

That's in community edition.  Even more control for Pro and Team customers. 🎉

**Comment by marklabrecque** [comment]
> Noting that latest release changes the behavior, and it just nags you instead, see [docker/roadmap#183 (comment)](https://github.com/docker/roadmap/issues/183#issuecomment-818910160) from thread above:
> 
> > [Y]ou are never forced to [update], you have the option to dismiss the popup until the next day.
> 
> That's in community edition. Even more control for Pro and Team customers. 🎉

The way the Docker Desktop app processes updates is very counter-intuitive. I have been getting many reports of this app auto-updating on their dev machines, so I have to test it out for myself. On my Macbook, I installed Lando (which came with the correct version of Docker ie 3.1) for the first time and setup my first application using Lando. Once everything was setup, I shut down the containers and shut down the Docker Desktop app from the menu bar, saying "Quit Docker Desktop". Then I restarted Docker Desktop and it showed me a dialog saying it was "Verifying Docker..." which happens when you install a new app. After that it asked for admin permissions to start, which it shouldn't need at this point. Low and behold, when everything was back up and running, it was running a newer version of Docker, 3.3.1, a version which is outside of the supported range for Lando.

If this isn't considered auto-updating, I don't know what is. I'm going to post in the thread you linked above, because this is a really unsettling policy.

To be clear, I'm not complaining about Lando -- I realize and understand that this is completely out of the scope of your project. I merely commented here in case other people come into this thread looking for solutions.

On our team, we are opting into the pre-release version  of Lando which as of now supports Docker 3.3.1, but why are we being forced to update to software we don't want to?
